### DISCUSSION TITLE
Trouble connecting to Mirth database

### DISCUSSION THREAD

**Discussion by Coooz** [nextgenhealthcare/connect#5282] [Q&A] [answered]
Hi,

I just installed Mirth 3.12 with a customer of ours, and now I have to replace the Derby database with a SQL Server database. Somehow I don't succeed; I get a message that the Mirth Connect service could not be started. 
This is a snippet of SQL Server Manager Studio that shows the Mirth_oru_processing database I would like to connect to.
![image](https://user-images.githubusercontent.com/88899312/178770164-a5995df9-3272-4d9b-8be8-f5c7e364f517.png)

I manage to log into SQL Server Manager Studio as follows:
![image](https://user-images.githubusercontent.com/88899312/178766535-600bc9dc-e608-4b18-94d1-77cda1698206.png)

Now I just need to fill out the Database tab of the Mirth Connect Server Manager (and then restart the service):
![image](https://user-images.githubusercontent.com/88899312/178768993-edf91caf-a907-4e05-a5ed-d491dfc67f7a.png)
I am pretty sure "Type", "Username" and "Password" are correct, but the URL is not.
These apparently are also incorrect:

```
jdbc:jtds:sqlserver://zrgdb06\inst03;databaseName=Mirth_oru_processing
jdbc:jtds:sqlserver://zrgdb06\inst03/Mirth_oru_processing
jdbc:jtds:sqlserver://zrgdb06:1433/Mirth_oru_processing
jdbc:jtds:sqlserver://zrgdb06/Mirth_oru_processing
```

I should mention that:
- I have copied the options like it says in point 4 of readme.txt, since neglecting this in the past resulted in not being able to start the Mirth Connect service as well.
- I am able to construct a regular SQL Server connection string that works just fine:
`Provider=SQLOLEDB.1;Password=XXXXXXXXXX;Persist Security Info=True;User ID=MyLoginName;Initial Catalog=Mirth_oru_processing;Data Source=zrgdb06\inst03`

What is the correct URL here - or what am I overlooking?

**Comment by jonbartels** [comment, marked as answer]
Stop using JTDS. MS has a new JDBC driver that is more compatible with newer releases of SQL Server and newer authentication protocols.

References:
* https://github.com/nextgenhealthcare/connect/search?q=jtds&type=discussions
* https://github.com/nextgenhealthcare/connect/discussions/5241

**Reply by Coooz** [reply]
Thanks jonbartels. It is working now. 
JTDS seems really outdated indeed. It would be nice if this wasn't the default option for sqlserver any longer.
